### ISSUE TITLE
Typing error with 1.0.3

### ISSUE THREAD

**Issue by bard** [redux-utilities/reduce-reducers#32]
Sorry to bring bad news again.

`tsconfig.json` and `index.ts` same as #30, new error:

```
node_modules/reduce-reducers/index.d.ts:7:1 - error TS1046: A 'declare' modifier is required for a top level declaration in a .d.ts file.

7 function reduceReducers<S>(initialState: S | null, ...reducers: Reducer<S>[]): Reducer<S>;
  ~~~~~~~~


Found 1 error.
```

**Comment by timche** [comment]
😞 I've released [v1.0.4](https://github.com/redux-utilities/reduce-reducers/releases/tag/v1.0.4)

**Comment by bard** [comment]
No sad face please! I'm grateful for you maintaining reduce-reducers, my redux-agent library depends on it and not just in a package.json sense. 

Just re-tested, everything working fine, thanks for the speedy fix! :+1:
